### PR TITLE
Loc fix for angle brackets in titles and headers

### PR DESCRIPTION
**Pull request by v-alje**
LOC is having trouble consuming angle brackets when they occur in metadata titles or headers. This fix is intended to make Loc and English render the same.

This bulk fix changes all "\&lt;" to "\<" and "\&gt;" to "\>" in metadata titles, and "\&lt;" to "\\\<" and "\&gt;" to ">" in section headers.

Fixes CPS User Story 62348
Related to #10115

### CODE REVIEW

**Comment by v-alje** [comment]
Should this be merged into another branch to get a preview build?

**Comment by mairaw** [comment]
Not needed @v-alje. You can preview just with the PR as well: https://review.docs.microsoft.com/en-us/dotnet/csharp/codedoc?branch=pr-en-us-10137#summary

**Comment by mairaw** [comment]
Question though: why isn't the change the same as the one that was done for #10115? We usually just escape the < and not the >.

**Comment by v-alje** [comment]
@mairaw I escaped the closing angle bracket to make certain there wouldn't be a chance that it could accedently close any previous open angle bracket. I can change it if you would like me to.

**Comment by mairaw** [comment]
I'd prefer @v-alje.

**Comment by v-alje** [comment]
@mairaw I have re-synced from master and re-run the bulk search and replace. to fix the closing angle brackets as requested.

File: docs/visual-basic/language-reference/error-messages/default-property-propertyname1-conflicts-with-default-property-propertyname2.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Default property &#39;&lt;propertyname1&gt;&#39; conflicts with default property &#39;&lt;propertyname2&gt;&#39; in &#39;&lt;classname&gt;&#39; and so should be declared &#39;Shadows&#39;"
+title: "Default property &#39;<propertyname1>&#39; conflicts with default property &#39;<propertyname2>&#39; in &#39;<classname>&#39; and so should be declared &#39;Shadows&#39;"
```
Comment (mairaw) [inline docs/visual-basic/language-reference/error-messages/default-property-propertyname1-conflicts-with-default-property-propertyname2.md]: @v-alje and what about the `&#39;`? Keep that like that?

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look good to me @v-alje but it seems we would also like to change that other encoded character I just mentioned.

File: docs/visual-basic/language-reference/error-messages/default-property-propertyname1-conflicts-with-default-property-propertyname2.md (hunk 1 above)
Comment (v-alje) [reply docs/visual-basic/language-reference/error-messages/default-property-propertyname1-conflicts-with-default-property-propertyname2.md]: @mairaw single quotes have been flattened as well now.

**Review by mairaw — APPROVED** [review]
Thank you for making the additional fixes @v-alje!

**Pull request merged**